pandas/tests/io/test_stata.py

```diff
@@ -1505,3 +1505,20 @@ def test_unicode_dta_118(self):
         expected = pd.DataFrame(values, columns=columns)
 
         tm.assert_frame_equal(unicode_df, expected)
+
+    def test_mixed_string_strl(self):
+        # GH 23633
+        output = [
+            {'mixed': 'string' * 500,
+             'number': 0},
+            {'mixed': None,
+             'number': 1}
+        ]
+
+        output = pd.DataFrame(output)
```

Comment:
is it worth having a test for other versions?

Reply:
Not relevant for the other version (114) which doesn't support strls.